packages/contracts/contracts/core/DAO.sol

```diff
@@ -196,10 +196,14 @@ contract DAO is IDAO, Initializable, UUPSUpgradeable, ACL, ERC1271, AdaptiveERC1
         _handleCallback(msg.sig, msg.data); // WARN: does a low-level return, any code below would be unreacheable
     }
 
+    /// @notice Emits the MetadataSet event if new metadata is set
+    /// @param _metadata Hash of the IPFS file object
```

Comment:
do we need `file` ?
`... IPFS object` might be enough.

Reply:
```suggestion
    /// @param _metadata Hash of the IPFS metadata object
```